code/game/objects/items/weapons/power_cells.dm

```diff
@@ -34,6 +34,9 @@
 	update_icon()
 	return 1
 
+/obj/item/weapon/cell/proc/get_charge_percent()
```

Comment:
This proc already exists, 
/obj/item/weapon/cell/proc/percent()		

Located in power/cell.dm
A lot of the core functions for power cells are there
use the existing one instead of duplicating it

Reply:
removed it